newIDE/app/src/KeyboardShortcuts/KeyboardShortcuts.spec.js

```diff
@@ -0,0 +1,100 @@
+import {
+  getShortcutStringFromEvent,
+  isValidShortcutEvent,
+  getElectronAccelerator,
+} from './index';
+
+/**
+ * Creates a KeyboardEvent-like object for testing
+ * functions that take event objects as input
+ */
+const keyEvent = (ctrlKey, shiftKey, altKey, code) => ({
```

Comment:
Uber nitpicking: I usually make functions to always start with a verb, because it's something that "do" something.
Here: `makeKeyEvent`

Reply:
My bad - fixed.